hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -20,6 +20,10 @@ parameters:
   value: "[]"
 - name: DELETE_PROTECTION
   value: ""
+- name: BACKUP_VELERO_ENABLED
+  value: false
```

Comment:
we might need double quotes here

cc @dustman9000 @jharrington22 

I've seen the pipeline complain about this in the past, on c-i-o stuff that Eric and Candace were working on

Reply:
Nice catch @gregsheremeta `oc template` doesn't like the non quoted value.

```
error: failed to read input object (not a Template?): unable to decode "./hack/olm-registry/olm-artifacts-template.yaml": v1.Template.Parameters: []v1.Parameter: v1.Parameter.Value: ReadString: expects " or n, but found f, error found in #10 byte of ...|,"value":false},{"na|..., bigger context ...|lue":""},{"name":"BACKUP_VELERO_ENABLED","value":false},{"name":"BACKUP_VELERO_NAMESPACE","value":"o|...
```